apps/researcher/src/lib/api/objects/rdf-helpers.ts

```diff
@@ -104,27 +105,29 @@ export function createPlaces(resource: Resource, propertyName: string) {
   return places.length > 0 ? places : undefined;
 }
 
-function fromStringToDate(dateValue: string | undefined) {
-  // TBD: change 'Date' to string to allow for uncertain dates (e.g. EDTF)?
-  let date: Date | undefined;
+function fromStringToEdtf(dateValue: string | undefined) {
   if (dateValue !== undefined) {
-    date = new Date(dateValue);
+    try {
+      return edtf(dateValue);
+    } catch (err) {
+      // Ignore invalid dates
+    }
   }
 
-  return date;
+  return undefined;
 }
 
 export function createTimeSpan(timeSpanResource: Resource) {
   const rawStartDate = getPropertyValue(timeSpanResource, 'ex:startDate');
-  const startDate = fromStringToDate(rawStartDate);
+  const startDate = fromStringToEdtf(rawStartDate);
 
   const rawEndDate = getPropertyValue(timeSpanResource, 'ex:endDate');
-  const endDate = fromStringToDate(rawEndDate);
+  const endDate = fromStringToEdtf(rawEndDate);
 
   const timeSpan: TimeSpan = {
     id: timeSpanResource.value,
-    startDate,
-    endDate,
+    startDate: startDate !== undefined ? new Date(startDate.min) : undefined, // E.g. 1881 = 1881-01-01
```

Comment:
Can we keep it an edtf date? It would be nice to show the date in the frontend as given by the museum, so:

1881 = 1881 
1881-01-01 = 1881-01-01

But this conversion removes the possibility to render the date as given by the museum.

Reply:
Decided offline: We will keep it as a date, and we will have a separate conversation about the formatting of dates

Reply:
Small addition: also find out whether there is a need for the API to keep working with Date objects. If we need to do date-time processing (e.g. sorting), dates-as-strings will not suffice. But if there is no necessity, now or in the future, the API might as well simply expose the dates as strings.